frontend/src/pages/RequestorPage/ResponsePanel.tsx

```diff
@@ -660,3 +660,13 @@ function FailedWebsocket() {
     </div>
   );
 }
+
+const safeParseJson = (jsonString: string) => {
```

Comment:
There's already an ~eerily similar~ exact same `safeParseJson` function in `src/utils/index.ts`. Why not import & use that?